### PR TITLE
Full doc-en render test

### PR DESCRIPTION
**Pull request by haszi**
This could be an additional test for testing PhD with a full render of the `doc-en` repo.

### CODE REVIEW

File: .github/workflows/full_render_doc-en_test.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    name: doc-en full render test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout PhD
+        uses: actions/checkout@v2
```
Comment (Girgias) [inline .github/workflows/full_render_doc-en_test.yml:9]: Normal this uses the V2 version?

**Review by Girgias — APPROVED** [review]
I think this makes sense to me, I don't really know GitHub actions, but this looks ok

File: .github/workflows/full_render_doc-en_test.yml (hunk 1 above)
Comment (haszi) [reply .github/workflows/full_render_doc-en_test.yml]: You're right, it should probably be v4. I copied it from an other test. :-)

**Comment by haszi** [comment]
I also renamed the workflow to `Render test` and the test to just `doc-en`, and added the output directory to the PhD call.

**Review by Girgias — APPROVED** [review]

**Pull request merged**